.github/workflows/node.js.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
As far as I can tell, we only need to run `test` and don't need to integrate with saucelabs?

Reply:
You might want to check out [this cache action](https://github.com/adobe/alloy/blob/37f94b60e64bad1f2950c9df527b5841287f738b/.github/workflows/test-cron.yaml#L14-L18) to optimize the installation of dependencies. I don't know a lot about it, but it seems nice.